Command/DebugBlocksCommand.php

```diff
@@ -14,7 +14,7 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\OptionsResolver\OptionsResolver;
+use Sonata\BlockBundle\Util\OptionsResolver;
```

Comment:
Don't know why this class exists. IMHO we should deprecate our `OptionsResolver` class. Can you do this @pestaa ?